base/time/time_unittest.cc

```diff
@@ -11,7 +11,10 @@
 #include <limits>
 #include <string>
 
+#if defined(STARBOARD)
```

Comment:
Let's consolidate this starboard check with the one below and put them all with the platform check below